let a = 0
let b = 3
try {
    console.log(vivel)
}
catch (err) {
    console.log( "this is error :"+err)
} 

finally{
    console.log("this is always run")
}